contracts/src/factory/facets/architect/ArchitectBase.sol

```diff
@@ -89,6 +89,14 @@ abstract contract ArchitectBase is Factory, IArchitectBase {
       _deployEntitlement(ims.ruleEntitlement, spaceAddress)
     );
 
+    // Confirm that deployed RuleEntitlement is expected version.
```

Comment:
Inserted this as a sanity check that we don't accidentally call either endpoint with the wrong contract, since the V1 contract cannot properly decode v2 rule data upon space creation